src/desktop/apps/sitemaps/test/routes.coffee

```diff
@@ -24,20 +24,19 @@ describe 'Sitemaps', ->
       @res = set: sinon.stub(), send: sinon.stub()
 
     afterEach ->
-      routes.__set__ 'NODE_ENV', 'test'
+      routes.__set__ 'ENABLE_WEB_CRAWLING', false
 
-    it 'renders a noindex in anything but production', ->
-      routes.__set__ 'NODE_ENV', 'staging'
+    it 'renders a noindex when ENABLE_WEB_CRAWLING is false', ->
+      routes.__set__ 'ENABLE_WEB_CRAWLING', false
       routes.robots null, @res
       @res.send.args[0][0]
         .should.equal  'User-agent: *\nNoindex: /'
 
-    describe 'production', ->
+    describe 'when ENABLE_WEB_CRAWLING is true', ->
       beforeEach ->
-        routes.__set__ NODE_ENV: 'production', APP_URL: 'https://www.artsy.net'
+        routes.__set__ ENABLE_WEB_CRAWLING: true, APP_URL: 'https://www.artsy.net'
```

Comment:
This should be a string to accurately reflect how production behaves, right?

Reply:
No, this is using `__set__`, which is Rewire stubbing the actual `ENABLE_WEB_CRAWLING` sharify key used in the file.

And that, when combined with the defaults/ENV values, _already_ gets transformed: https://github.com/artsy/force/blob/a12a301e0a1941b50789cf0ebdf29ded7313b679/src/desktop/config.coffee#L122

Reply:
So stubbing at this point, you should be stubbing the actual values ^ (from `JSON.parse`)

Reply:
Cool. Didn't know we did `JSON.parse`!